release_notes/372-release-notes.adoc

```diff
@@ -236,12 +236,24 @@ Kubernetes users must disable the admission controller plugin for PSPs before up
 [id="bug-fixes-372"]
 == Bug fixes
 
-* Before this update, the steps to configure {ocp} OAuth for more than one URI were missing. The documentation has been revised to include instructions for configuring OAuth in {ocp} to use more than one URI. For more information, see xref:../operating/manage-user-access/configure-ocp-oauth.adoc#create-additional-routes-ocp-oauth_configure-ocp-oauth[Creating additional routes for the OpenShift Container Platform OAuth server]. (ROX-11296) 
+* Before this update, the steps to configure {ocp} OAuth for more than one URI were missing. The documentation has been revised to include instructions for configuring OAuth in {ocp} to use more than one URI. For more information, see xref:../operating/manage-user-access/configure-ocp-oauth.adoc#create-additional-routes-ocp-oauth_configure-ocp-oauth[Creating additional routes for the OpenShift Container Platform OAuth server]. (ROX-11296)
 * Before this update, the autogenerated image integration, such as a Docker registry integration, for a cluster is not deleted when the cluster is removed from Central. This issue is fixed. (ROX-9398)
 * Before this update, the `Image OS` policy criteria did not support regular expressions, or `regex`. However, the documentation indicated that regular expressions were supported. This issue is fixed by adding support for regular expressions for the `Image OS` policy criteria. (ROX-12301)
 * Before this update, the syslog integration did not respect a configured TCP proxy. This is now fixed.
 * Before this update, the `scanner-db` pod failed to start when a resource quota was set for the `stackrox` namespace, because the `init-db` container in the pod did not have any resources assigned to it. The `init-db` container for `ScannerDB` now specifies resource requests and limits that match the `db` container. (ROX-12291)
 
+[id="resolved-in-version-3721"]
+=== Resolved in version 3.72.1
+
+Release date: October 13, 2022
+
+* Before, because of a bug in {product-name-short} 3.72.0, the Collector pods stopped responding and reach a segmentation fault after allocating a memory block for the protocol buffer heap under certain load conditions.
```

Comment:
You could reduce the number of commas in the first sentence by updating it to something like the following:

`* Because of a bug in {product-title-short} 3.72.0, the Collector pods previously stopped responding and reach a segmentation fault after allocating a memory block for the protocol buffer heap under certain load conditions.`